src/main/resources/addresources/values/strings.xml

```
@@ -13,14 +13,17 @@
             <string name="revanced_settings_import_failure_parse">Import failed: %s</string>
         </patch>
         <patch id="misc.gms.BaseGmsCoreSupportResourcePatch">
+            <string name="gms_core_dialog_title">Action needed</string>
+            <string name="gms_core_dialog_not_whitelisted_not_allowed_in_background_message">MicroG GmsCore does not have permission to run in the background.\n\nFollow the \"Don\'t kill my app\" guide for your phone, and apply the instructions to your MicroG installation.\n\nThis is required for this app to work.</string>
+            <string name="gms_core_dialog_open_website_text">Open website</string>
+            <string name="gms_core_dialog_not_whitelisted_using_battery_optimizations_message">MicroG GmsCore battery optimizations must be turned off to prevent app crashes.\n\nPress the continue button, and allow turning off battery optimizations.</string>
```

Turned off -> Disabled (Probably a simpler term)
App crashes -> Issues (Crashes arent the only issue, playback could also silently fail apparently)
```suggestion
            <string name="gms_core_dialog_not_whitelisted_using_battery_optimizations_message">MicroG GmsCore battery optimizations must be disabled to prevent issues.\n\nPress the continue button and disable battery optimizations.</string>
```